docs/source/pythonapi/examples/mdgxs-part-i.ipynb

```diff
@@ -826,7 +833,7 @@
        "      <th>199</th>\n",
        "      <td>1</td>\n",
        "      <td>1</td>\n",
-       "      <td>1</td>\n",
+       "      <td>0</td>\n",
```

Comment:
This (and all related comments) are now fixed.  The problem was that 1) I should have been adding one to the group index after I brute-forced it, and 2) the `np.searchsorted` method I used actually resulted in a bin/group value of up to 1 higher than it should have been due to floating point error (the difference between values written to a statepoint, then read-back in and those which were generated and used on the python side only).

I have a much generic method now which saves alot of lines and maybe will lead to some condensing of code/confusion later.